src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -19,7 +31,19 @@
      *                 В любом другом случае бросить DateTimeException
      **/
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+        final LocalDate NOW_LOCAL_DATE = LocalDate.now();
```

Comment:
don't make it final and fix travis
you got it right that this should be a local variable